lib/colorado_lottery.rb

```diff
@@ -0,0 +1,44 @@
+class ColoradoLottery
+
+  attr_reader :registered_contestants, :winners, :current_contestants
+
+  def initialize
+    @registered_contestants = {}
+    @winners = []
+    @current_contestants = {}
+  end
+
+  def interested_and_18?(contestant, game)
+    contestant.age >= 18 && contestant.game_interests.include?(game.name)
+  end
+
+  def can_register?(contestant, game)
+    interested_and_18?(contestant, game) && (contestant.out_of_state? == false || game.national_drawing? == true)
```

Comment:
Great job using the return value of `interested_and_18?` in conjunction with the new conditions.